### PR TITLE
chore: 🤖 improve issue template

### PR DESCRIPTION
**Pull request by IWANABETHATGUY**
### Description
1. remove some unnecessary config

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -11,24 +11,16 @@ body:
 
         Prerequisites
 
-        - Rust >= 1.75
         - Node.js >= 18.18.0
-        - Yarn >= 4
```
Comment (IWANABETHATGUY) [inline .github/ISSUE_TEMPLATE/bug_report.yml:16]: `rust` and `yarn` is not necessary for a user who use `rolldown`,  I am not sure what is the lowest nodejs version do we support?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -11,24 +11,16 @@ body:
...
         - Git
 
         - Read the [docs](https://rolldown.rs/)
-        - Ask on [Discord Chat](https://chat.rolldown.rs)
```
Comment (IWANABETHATGUY) [inline .github/ISSUE_TEMPLATE/bug_report.yml:20]: Not necessary for now.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -11,24 +11,16 @@ body:
...
+        # - Ask on [Discord Chat](https://chat.rolldown.rs)
         - Ask on [GitHub Discussions](https://github.com/rolldown/rolldown/discussions)
         
         Also try to search for your issue - it may have already been answered or even fixed in the development branch.
         However, if you find that an old, closed issue still persists in the latest version,
         you should open a new issue using the form below instead of commenting on the old issue.
-  - type: input
```
Comment (IWANABETHATGUY) [inline .github/ISSUE_TEMPLATE/bug_report.yml:26]: `npx envinfo --system --npmPackages rolldown --binaries --browsers` already print the version of `rolldown`

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
-contact_links:
```
Comment (IWANABETHATGUY) [inline .github/ISSUE_TEMPLATE/config.yml:2]: Not necessary for now.

**Review by hyf0 — APPROVED** [review]

**Pull request merged**